parent/pom.xml

```diff
@@ -133,11 +133,37 @@
                         <arguments>-Poss-release ${release.arguments}</arguments>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-deploy-plugin</artifactId>
+                    <configuration>
+                        <!-- To prefer nexus-staging-maven-plugin -->
+                        <skip>true</skip>
+                    </configuration>
+                </plugin>
+                <plugin>
+                    <groupId>org.sonatype.plugins</groupId>
+                    <artifactId>nexus-staging-maven-plugin</artifactId>
+                    <version>1.6.8</version>
```

Comment:
Original idea of parent pom was to not put vertsions here and leave them on child projects to set (see oss-release profile). Are we going to change this?

Reply:
Most projects don't need to configure anything with this plugin.
Not setting the version would require every project to have a pluginManagement section just to set the version.

Reply:
I understand, but that's exactly how oss-release profile was made. Having versions here means we have to update the parent pom more frequently.
Each option has it's advantages, but also cost something.

Reply:
@Tomas-Kraus I don't think it's going to be changed. It's more about using it when it makes sense only. Ie for javadoc plugin it does not make much sense as that is something project need to define anyway depending (not only) on its source level setting. As long as version 1.6.8 runs properly on Java SE 11 (nice to have; 8 is a must) to avoid early update, I'm fine with this.

Reply:
Well, in case of oss-release I had to add several plugin versions in every single project I was adopting for eclipse to avoid warnings. So I see no difference between main plugin management section and this profile. It would be good to go one way, maybe, with versions for everything.